### DISCUSSION TITLE
Laravel livewire binding checkboxes from array of object

### DISCUSSION THREAD

**Discussion by dennykurniawan48** [livewire/livewire#8432] [1. Help] [answered]
How to bind an array of objects to checkboxes in Laravel Livewire:

The array looks like this:

`[{"name":"A","selected":true}, {"name":"B","selected":true}]`
My view blade:

`<input type="checkbox" wire:model="list.{{$index}}.selected">`

The problem is when the checkbox is selected on unselected the array is not updated, how to solve this?

**Comment by joshhanley** [comment, marked as answer]
@dennykurniawan48 try `wire:model.live`

**Reply by dennykurniawan48** [reply]
Thank you Livewire Developer, it is working like a charm.
